server/routes/config.api.js

```diff
@@ -182,6 +182,8 @@ if (Object.keys(json).length !== 0) {
   config_current.output = json.output || config_current.output;
   config_current.city = json.city || config_current.city;
   config_current.state = json.state || config_current.state;
+  config_current.country = json.country || config_current.country;
```

Comment:
These 2 are unneccesry, only for importing variables from the old bash version config <v1.0 a list of them can be found here https://github.com/liam8888999/ErsatzTV-Filler/blob/bash-version/sample-config.conf and if they are not in there it cant be translated to json format config anyway

Reply:
Removed